dev-db/sqlitestudio/sqlitestudio-3.4.3.ebuild

```diff
@@ -99,7 +117,7 @@ src_prepare() {
 
 	local ts_dir_main="SQLiteStudio3/sqlitestudio/translations"
 	plocale_find_changes ${ts_dir_main} "sqlitestudio_" '.ts'
-	plocale_for_each_locale prepare_locale
+	plocale_for_each_locale lrelease_locale
 	plocale_for_each_disabled_locale rm_locale
 
 	# prevent "multilib-strict check failed" with USE test
```

Comment:
This does not work properly, the tests fail with exactly that error:
```
scanelf: rpath_security_checks(): Security problem with relative DT_RUNPATH '.' in /var/tmp/portage/dev-db/sqlitestudio-3.4.3/image/usr/lib64/libcoreSQLiteStudio.so.1.0.0
...
scanelf: rpath_security_checks(): Security problem with relative DT_RUNPATH '.' in /var/tmp/portage/dev-db/sqlitestudio-3.4.3/image/lib/libTestUtils.so.1.0.0
Files matching a file type that is not allowed:
   lib/libTestUtils.so.1.0.0
```

Please investigate what's needed to fix the tests